src/plugins/intel_cpu/src/cpu_streams_calculation.hpp

```diff
@@ -15,10 +15,13 @@ namespace ov {
 namespace intel_cpu {
 /**
  * @brief      Generate streams information table according to processors type table
- * @param[in]  input_streams is target streams set by user via NUM_STREAMS or hints.
+ * @param[in]  input_streams is target streams set by user via ov::num_streams or hints.
  *               - input "0" mean function generate the optimal number of streams
- *               - LATENCY hint equals 1 stream.
- * @param[in]  input_threads is max threads set by user via INFERNECE_NUM_THREADS.
+ *               - LATENCY hint equals 1 stream on one socket platform.
+ * @param[in]  input_threads is max threads set by user via ov::inference_num_threads.
```

Comment:
input_threads is the max number of threads set by user...

Reply:
updated